src/deepsparse/yolo/annotate.py

```diff
@@ -114,15 +111,6 @@
     help="Inference engine backend to run on. Choices are 'deepsparse', "
     "'onnxruntime', and 'torch'. Default is 'deepsparse'",
 )
-@click.option(
-    "--image_shape",
-    "--image-shape",
-    type=int,
-    nargs=2,
-    default=(640, 640),
-    help="Image shape to use for inference, must be two integers",
-    show_default=True,
-)
```

Comment:
i personally would still like to change the image shape of the model/pipeline as this can affect accuracy and performance

Reply:
This argument really wasn't doing anything [as in the model shape wasn't being overidden]; it was only being used to calculate scales which the pipeline does implicitly now; 

if we need support for model shape overiding I think that should be a separate PR. Does that make sense?

Reply:
@markurtz

Reply:
@rahul-tuli this is a bug then - the `image-shape` arg should be passed to the pipeline as the shape we run inference at. let's get this added back in, with image shape propagated correctly

Reply:
Support Added Here: https://github.com/neuralmagic/deepsparse/pull/883

Reply:
Support merged!